logger/dapr_logger.go

```diff
@@ -0,0 +1,146 @@
+package logger
+
+import (
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// daprLogger is the implemention for logrus
+type daprLogger struct {
+	// name is the name of logger that is published to log as a scope
+	name string
+	// loger is the instance of logrus logger
+	logger *logrus.Entry
+}
+
+var GetVersion = func() string {
+	return "unknown"
+}
+
+func newDaprLogger(name string) *daprLogger {
+	newLogger := logrus.New()
+	newLogger.SetOutput(os.Stdout)
+
+	dl := &daprLogger{
+		name: name,
+		logger: newLogger.WithFields(logrus.Fields{
+			logFieldScope: name,
+			logFieldType:  LogTypeLog,
+		}),
+	}
+
+	dl.EnableJSONOutput(defaultJSONOutput)
+
+	return dl
+}
+
+// EnableJSONOutput enables JSON formatted output log
+func (l *daprLogger) EnableJSONOutput(enabled bool) {
```

Comment:
Please, get the version as param instead.

Reply:
The problem with is that is called by `newDaprLogger` which is not exposed by the package. I propose keeping this as a global variable (change from func to a string variable) to keep the changes in dapr/dapr to a minimum. There is a much more elegant way of handling this but it is outside the scope of this PR.

Reply:
I see it. OK, we can refactor that later.